controller-server/src/main/java/com/yahoo/vespa/hosted/controller/restapi/application/DeployAuthorizer.java

```diff
@@ -43,36 +39,36 @@ public void throwIfUnauthorizedForDeploy(Principal principal,
                                              Environment environment,
                                              Tenant tenant,
                                              ApplicationId applicationId) {
-        if (athenzCredentialsRequired(environment, tenant, applicationId, principal))
-            checkAthenzCredentials(principal, tenant, applicationId);
-    }
-
-    // TODO: inline when deployment via ssh is removed
-    private boolean athenzCredentialsRequired(Environment environment, Tenant tenant, ApplicationId applicationId, Principal principal) {
-        if (!environmentRequiresAuthorization(environment))  return false;
-
-        if (! isScrewdriverPrincipal(principal))
-            throw loggedForbiddenException(
-                    "Principal '%s' is not a screwdriver principal, and does not have deploy access to application '%s'",
-                    principal.getName(), applicationId.toShortString());
+        if (!environmentRequiresAuthorization(environment))  return;
 
-        return tenant.isAthensTenant();
-    }
+        if (principal == null) {
+            throw loggedUnauthorizedException("Principal '%s' is not authenticated.", principal.getName());
+        }
 
+        if (!(principal instanceof AthenzPrincipal)) {
+            throw loggedUnauthorizedException(
+                    "Principal '%s' of type '%s' is not an Athenz principal, which is required for production deployments.",
+                    principal.getName(), principal.getClass().getSimpleName());
+        }
 
-    // TODO: inline when deployment via ssh is removed
-    private void checkAthenzCredentials(Principal principal, Tenant tenant, ApplicationId applicationId) {
-        AthenzDomain domain = tenant.getAthensDomain().get();
-        if (! (principal instanceof AthenzPrincipal))
-            throw loggedForbiddenException("Principal '%s' is not authenticated.", principal.getName());
+        AthenzPrincipal athenzPrincipal = (AthenzPrincipal) principal;
+        AthenzDomain principalDomain = athenzPrincipal.getDomain();
 
-        AthenzPrincipal athensPrincipal = (AthenzPrincipal)principal;
-        if ( ! hasDeployAccessToAthenzApplication(athensPrincipal, domain, applicationId))
+        if (!principalDomain.equals(AthenzUtils.SCREWDRIVER_DOMAIN))
```

Comment:
There seems to be a mix of braces/no braces for single statement blocks in this method now. I suggest just using braces for all cases.

Reply:
I will add curly braces on all statement to be consistent.